api/pom.xml

```diff
@@ -46,6 +46,11 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+         <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.annotation.versioning</artifactId>
+            <version>1.0.0</version>
+        </dependency>
```

Comment:
@Emily-Jiang to mark this as a provided or optional dependency.

Reply:
mark the dependency as provided.